sites/industrial-lasers/server/templates/website-section/index.marko

```diff
@@ -33,7 +33,7 @@ $ const adSlots = {
     <div class="col-lg-8">
       <endeavor-content-query-feed
         header="Featured"
-        limit=7
+        limit=8
         skip=5
```

Comment:
You'll also need to increase the `skip=12` on L51 to `skip=13` since this limit increased.